packages/paste-core/components/button/__tests__/button.test.tsx

```diff
@@ -78,13 +78,6 @@ describe('Button', () => {
           </Button>
         )
       ).toThrow();
-      expect(() =>
```

Comment:
question: why remove this?

Reply:
We now allow <Button as='a' variant="inverse"> for the external link button